content/en/docs/concepts/specs-requirements.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Notary v2 project specifications and requirements"
+description: "Notary v2 project specifications and requirements"
+type: docs
+weight: 4
+---
+
+More information on the specifications, requirements, and other details about the Notary v2 project can be found in the [Notary v2 repository](https://github.com/notaryproject/notaryproject/tree/main).
```

Comment:
`Notary v2` ==> `Notary`, since we don't use/reference `Notary v2` anymore.

Reply:
fixed